backend/app/routes/joinUs/@validationSchema/index.js

```diff
@@ -5,8 +5,6 @@ const JoinUsValidationSchema = Joi.object().keys({
   email: Joi.string().required(),
   linkdin: Joi.string().required(),
   description: Joi.string().required(),
-  domain: Joi.array().required(),
```

Comment:
This is different from the `otherDomain` field so please don't remove it.

Reply:
`domain` duplicates `interestedDomain`

Reply:
Ah, correct! I misse this field :)